src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-form/cephfs-form.component.html

```diff
@@ -15,6 +15,26 @@
                         i18n
                         *ngIf="!editing">Orchestrator is not configured. Deploy MDS daemons manually after creating the volume.</cd-alert-panel>
       </ng-container>
+
+      <cd-alert-panel type="info"
+                      class="m-3"
+                      spacingClass="mt-3"
+                      i18n
+                      *ngIf="editing && disableRename">
+        <p>File System can only renamed if it is shutdown and `refuse_client_session` is set to true.
+           You need to do the following in command line and refresh the page:</p>
+        <p>{{ fsFailCmd }}
```

Comment:
Some text suggestion

```suggestion
        <p>The File System can only be renamed if it is shutdown and `refuse_client_session` is set to true.
           Follow the steps below in the command line and refresh the page:</p>
        <p>{{ fsFailCmd }}
```

Reply:
done, thanks!